File: 4-Patron/php/InfoSkin.php
<?php
class InfoSkin
{
  private $autor;
  private $fechaCreacion;
  private $descripcion;

  function __construct($autor, $fecha, $descripcion)
  {
    $this->autor = $autor;
    $this->fechaCreacion = $fecha;
    $this->descripcion = $descripcion;
  }

  function getAutor()
  {
    return $this->autor;
  }

  function setAutor($nAutor)
  {
    $this->autor = $nAutor;
  }

  function getFecha()
  {
    return $this->fechaCreacion;
  }

  function setFecha($nFecha)
  {
    $this->fechaCreacion = $nFecha;
  }

  function getDescripcion()
  {
    return $this->descripcion;
  }

  function setDescripcion($nDescripcion)
  {
    $this->descripcion = $nDescripcion;
  }

  function getInfoSkin()
  {
    return  [$this->autor, $this->fechaCreacion, $this->descripcion ];
  }
}
?>

File: 4-Patron/Java/InfoSkin.java
package patron4;

public class InfoSkin {
    private String Autor;
    private String fechaCreacion;
    private String Descripcion;

    public InfoSkin(String Autor, String fechaCreacion, String Descripcion) {
        this.Autor = Autor;
        this.fechaCreacion = fechaCreacion;
        this.Descripcion = Descripcion;
    }

    public String getAutor() {
        return Autor;
    }

    public String getDescripcion() {
        return Descripcion;
    }

    public String getFechaCreacion() {
        return fechaCreacion;
    }

    public void setAutor(String Autor) {
        this.Autor = Autor;
    }

    public void setDescripcion(String Descripcion) {
        this.Descripcion = Descripcion;
    }

    public void setFechaCreacion(String fechaCreacion) {
        this.fechaCreacion = fechaCreacion;
    }
    
    public String[] getInfoSkin(){
        String[] info  = new String[]{this.Autor,this.fechaCreacion,this.Descripcion};
        return info ;
    }
}
